tests/test_cli.py

```diff
@@ -130,6 +130,9 @@ def test_command_error_missing_services(self):
             with self.assertRaises(SystemExit) as cm:
                 self.cli_command(argv)
             self.assertEqual(cm.exception.code, 666)
+            self.logger_mock.critical.assert_called_with(
+                "Could not find all tracked services. Missing: stack_app, stack_db"
+            )
```

Comment:
Perhaps we should match what is written to `stdout`/`stderr` instead of intercepting/asserting on the logger?

Though I see that we already have a `logger_mock` and other tests following this approach, it's probably a larger change. Feel free to keep it as is if you want.

Reply:
I think pytest already does stuff to `stdout` so it might be hard, not sure. Skipping it for this PR though either way 😃